pkg/version/version.go

```diff
@@ -58,6 +61,11 @@ func GetISOPath() string {
 	return isoPath
 }
 
+// GetHyperKitVersion returns the specific hyperKit version
+func GetHyperKitVersion() string {
+	return hyperKitVersion
```

Comment:
Move the expected version into `drvs/hyperkit` - as it's only consumed there.

Reply:
I moved this constant and getter method.
Makefile(& pkg/version/version.go) → pkg/minikube/registry/drvs/hyperkit.go